RevenueCatUI/CustomerCenter/Views/CustomerCenterView.swift

```diff
@@ -28,24 +28,31 @@ public struct CustomerCenterView: View {
 
     @StateObject private var viewModel: CustomerCenterViewModel
 
+    private var appearance: CustomerCenterConfigData.Appearance
+
     /// Create a view to handle common customer support tasks
-    public init(customerCenterActionHandler: CustomerCenterActionHandler? = nil) {
+    public init(customerCenterActionHandler: CustomerCenterActionHandler? = nil,
+                appearance: CustomerCenterConfigData.Appearance = .default) {
```

Comment:
Same here as in the promotional offers PR. I wonder if we need a default `Appearance` or if we can just pass `nil` to the environment... However, I think this is better in this case since default to `.system` makes more sense than defaulting to a specific language.